### DISCUSSION TITLE
【咨询】当电脑时间偏移: 0.06852889060974121秒时抢票时间是00秒或者是01秒为宜？

### DISCUSSION THREAD

**Discussion by dimensioncity** [mikumifa/biliTickerBuy#370] [聊天专区]
【咨询】当电脑时间偏移: 0.06852889060974121秒时抢票时间是00秒或者是01秒为宜？（我怕提前抢票会被风控）

**Comment by dimensioncity** [comment]
另外我也想问一下可以先去其他（不是BW）已开票的展览中预填验证码吗？

**Comment by mikumifa** [comment]
1. 帮你修偏移了
2. 好像不太行

**Comment by gpc123456** [comment]
偏移正为延后抢票，负为提前抢票。（这个提前和延后是相较于你的电脑本地时间的）

这个偏移是使用阿里云提供时间服务自动校对进行补偿的，可以使抢票开始时间与GMT+8标准时间的误差小于10ms。

如果您担心触发风控，可以到抢票选项卡中，展开`手动设置/更新时间偏差`选项卡，将时间偏移稍微加上200-300ms，避免因为B站服务器时间不准造成被风控。

**Reply by dimensioncity** [reply]
ok阿里嘎多
